core/src/torii/mod.rs

```diff
@@ -364,6 +395,24 @@ async fn handle_subscription(events: EventsSender, stream: WebSocket) -> eyre::R
     Ok(())
 }
 
+/// Response body for get metrics request
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub struct Metrics {
+    /// Number of connected peers
+    peers: u64,
+    /// Number of committed blocks
+    blocks: u64,
+}
+
+#[allow(clippy::unused_async)]
+async fn handle_metrics<W: WorldTrait>(state: ToriiState<W>) -> Result<Json> {
+    let metrics: Metrics = Metrics {
+        peers: state.wsv.peers().len() as u64,
```

Comment:
We need to get connected peers from the p2p crate somehow. In WSV we have the general number of peers that this peer knows about, but they are not necessarily connected at the moment. If they are connected or not knows only our p2p lib.

Reply:
`iroha_p2p::network::NetworkBase::peers` has what you need.